torch/fx/graph.py

```diff
@@ -558,7 +558,14 @@ def register_modules_used(qualified_name : str):
                 modules_used.add(module_name)
 
         def type_repr(o : Any):
-            typename = _type_repr(o)
+            # Preprocessing for if we have a forward-declared string. 
+            # We want to make sure that 'torch.Tensor', "torch.Tensor",
+            # and torch.Tensor are all treated the same. We need to run
+            # `strip("'")` before and after `strip("\"")` to get rid of
+            # the additional quotes that `from __future__ import 
+            # annotations` adds
+            typename: str = _type_repr(o).strip("'").strip("\"").strip("'")
```

Comment:
Cool. Can we also look into:

1. Adding a test for things that aren't in `torch` to make sure that module imports work correctly
2. Make sure that the semantics here match what's desired in that linked PEP (I haven't read it). In particular, is there any guidance about the evaluation environment for these type annotations? e.g. if I have to go from string type annotation to actual type expression, what should be in my `globals`?